android/src/main/java/com/neuroidreactnativesdk/NeuroidReactnativeSdkModule.kt

```diff
@@ -99,6 +99,11 @@ class NeuroidReactnativeSdkModule(reactContext: ReactApplicationContext) :
         NeuroID.getInstance()?.setUserID(id)
     }
 
+    @ReactMethod
+    fun setRegisteredUserID(id: String) {
+        NeuroID.getInstance()?.setRegisteredUserID(id)
+    }
```

Comment:
This should return the native Boolean Result

Reply:
Are we changing setUserID to have this same behavior?

Reply:
Yes we should, I must have missed it when I changed to the boolean async/await style

Reply:
Android implementation is following the iOS implementation: 
```
fun setRegisteredUserId(registeredUserId: String): Boolean
fun getRegisteredUserId() = registeredUserID
```